AirCasting/CreateSessionViews/CreateSessionDetailsView.swift

```diff
@@ -47,54 +48,68 @@ struct CreateSessionDetailsView: View {
                 .frame(maxWidth: .infinity, minHeight: geometry.size.height, alignment: .top)
             }
         }
-                .simultaneousGesture(
-        
+        .simultaneousGesture(
             DragGesture(minimumDistance: 2, coordinateSpace: .global)
-                .onChanged({ (_) in
+                .onChanged { _ in
                     UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to: nil, from: nil, for: nil)
-                })
+                }
         )
     }
 }
 
 private extension CreateSessionDetailsView {
     var continueButton: some View {
         Button(action: {
+            showingAlert = false
```

Comment:
I think you don't have to do `showingAlert = false` here. You create it with an initial value so at this point it's still false. if we change it to true and then dismiss the alert it'll change the value back to false.